### DISCUSSION TITLE
Get-DbaFile error

### DISCUSSION THREAD

**Discussion by GregatBMW** [dataplat/dbatools#8504] [General] [answered]
Probably something I'm nott doing correctly:

Getting this:
WARNING: [09:36:44][Test-DbaPath] Error occurred while establishing connection to smuc13652.bmwgroup.net | The system cannot find the file specified

What I ran:
cls
$credential = Get-Credential
Test-DbaPath -SqlInstance smuc13652.bmwgroup.net -SqlCredential $credential -Path 'E$\Program Files\Microsoft Power BI Report Server\PBIRS\LogFiles'

Tried with E: in -Path results in same warning.

What am I missing?

**Comment by potatoqualitee** [comment]
Hey greg, can you use -Verbose and let us know?

**Reply by GregatBMW** [reply]
-Verbose did not seem to provide more info ??

PS C:\> $credential = Get-Credential -verbose
Test-DbaPath -SqlInstance smuc13652.bmwgroup.net -SqlCredential $credential -Path 'E:\Program Files\Microsoft Power BI Report Server\PBIRS\LogFiles' -Verbose

cmdlet Get-Credential at command pipeline position 1
Supply values for the following parameters:
WARNING: [12:37:54][Test-DbaPath] Error occurred while establishing connection to smuc13652.bmwgroup.net | The system cannot find the file specified

PS C:\>

**Comment by potatoqualitee** [comment, marked as answer]
That is surprising. Sometimes it says that when it cant connect at all. Can you try connecting with Connect-DbaInstance or Get-DbaDatabase? Including Verbose.

**Reply by GregatBMW** [reply]
Ah, maybe the issue is this is not a SQL server? It's a PowerBI FE server ?????

**Reply by GregatBMW** [reply]
An actual SQL server with Connect-DbaInstance

PS C:\> Connect-DbaInstance -SqlInstance GMUC12249G4.bmwgroup.net\PROD04 -Verbose

ComputerName Name                            Product              Version   HostPlatform IsAzure IsClustered ConnectedAs
------------ ----                            -------              -------   ------------ ------- ----------- -----------
GMUC12249G4  GMUC12249G4.bmwgroup.net\PROD04 Microsoft SQL Server 15.0.4198 Windows      False   True        MUC\AXZ1CQY

**Reply by GregatBMW** [reply]
That must be the issue as this returns true

PS C:\> 
$credential = Get-Credential -verbose
Test-DbaPath -SqlInstance GMUC12249G4.bmwgroup.net\PROD04 -SqlCredential $credential -Path 'C:\' -Verbose
cmdlet Get-Credential at command pipeline position 1
Supply values for the following parameters:
True

**Reply by GregatBMW** [reply]
I was just assuming a file system thing would work on a non-SQL server

**Reply by potatoqualitee** [reply]
OHHH it;s connecting using named pipes and the named pipe is the "file" that cannot be found!

No, unfortunately, it only works for SQL Servers -- using WinRM would make it not work easily or by default on Linux.
